options/locale/locale_en-US.ini

```diff
@@ -112,6 +112,10 @@ step2 = Step 2:
 error = Error
 error404 = The page you are trying to reach either <strong>does not exist</strong> or <strong>you are not authorized</strong> to view it.
 
+failure = Failure
+pending = Pending
```

Comment:
"Pending" is not a general word, it could have different translations.

So, these translations need prefixes IMO

Reply:
I think they are fine, the less duplicate translations, the better.

Reply:
I do not think it is right. For Chinese, there are different meanings according to different context:

![image](https://github.com/go-gitea/gitea/assets/2114189/64700e9e-e5cb-44d4-9dea-a2bbceef0ecb)

Reply:
Does it matter in Gitea context? I think everything "pending" is some kind of "task". Do you see more kinds of pending things that do not classify as such?

Reply:
I am not a fan of making things potential ambiguity, while I won't block it.

Reply:
Is this word only for commit status?

Reply:
> Is this word only for commit status?

`issues.review.pending = Pending`

Reply:
I am pretty sure that "pending" is not general.

For example:

```
# zh-CN
issues.review.pending=待定 (waiting to be determined)
# zh-TW
issues.review.pending=待處理 (waiting to be processed)
```

The zh-TW is better than zh-CN because the "pending" review are waiting to be submitted (to be processed).

Meanwhile, for a "commit status", it's more like "waiting to be determined" but not "waiting to be processed", because the "commit status" is not for "processing", it just gets a determined status after the task finishes.

Reply:
Done in https://github.com/go-gitea/gitea/pull/26326/commits/f10f512639c1237025bab9a13daed4c46f767493